### ISSUE TITLE
maybe rework with a seed.js

### ISSUE THREAD

**Issue by jeanmw** [SF-WDI-LABS/mongoose-associations#2]
The repl works, but Justin and I ended up teaching this with seed.js because the concept of repl was confusing to students. I think it would also set them up better for the books lab etc to use a seed file instead. @justincastilla

**Comment by bgveenstra** [comment]
agreed - node console.js is unfamiliar and requires extra setup time (or remove it)

**Comment by justincastilla** [comment]
Why do we have embedded referencing and then referencing? We need to include pure referencing. Ugh. embedded referencing. Gross.
